### DISCUSSION TITLE
Add MSM site information to "channel" filter in relationship field control

### DISCUSSION THREAD

**Discussion by jcogs-design** [ExpressionEngine/ExpressionEngine#2909] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
If you use a relationship field within an MSM site the "channel" drop down filter lists all the allowed channels but provides no information about which MSM site the channels are defined within. There is no easy way to determine which site identically named channels relate to... in the example shown in image there is no way of knowing which MSM sites the two "information" channels relate to.

![Screenshot 2021-09-29 at 11 11 28](https://user-images.githubusercontent.com/13821249/135249763-9cbd1a81-74a6-4afa-8f88-ef62b9dcae4e.png)

**Describe the solution you'd like**
To add a sub-line to each entry when there are MSM sites defined giving the site the channel is defined within.

**Describe alternatives you've considered**
Adding the name of the MSM site to the channel - which is heavy handed solution for a small issue.

**Teachability, Documentation, Adoption, Migration Strategy**
The utility of the additional information would be self-evident and not require any significant documentation. The change would be backward compatible.

- [ ] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by intoeetive** [comment]
@ajohnson @Yulyaswan bringing this to your attention as something to consider as part of changes you're working on

**Comment by robsonsobral** [comment]
There's [my PR](https://github.com/ExpressionEngine/ExpressionEngine/pull/1545) starting to work on it. If you could give us the minimum, before the optimum, I would appreciate.
